Source/WTF/wtf/CurrentTime.cpp

```diff
@@ -291,7 +295,7 @@ MonotonicTime MonotonicTime::now()
     };
     static const tbClock tb;
     return fromRawSeconds(static_cast<double>(__builtin_ppc_get_timebase()) / tb.clockFrequency());
-#elif OS(LINUX) || OS(FREEBSD) || OS(OPENBSD) || OS(NETBSD)
+#elif OS(LINUX) || OS(FREEBSD) || OS(OPENBSD) || OS(NETBSD) || OS(AMIGAOS)
```

Comment:
Just a discussion regarding Time "stuff" in CLIB2 vs Newlib. Wondering if it is worth us coming back to this with the knowledge that Newlib may not have this. We may need to also check if we are CLIB2 as well as AmigaOS.

Reply:
If newlib lacks something this will be exposed when we will start using it. I tried a couple of times and breaks in different places. So for now I focus on the new clib2 and as soon as the code compiles we can try to use newlib as well.

Reply:
okay, that sounds good.